sklearn/metrics/cluster/_bicluster.py

```diff
@@ -45,6 +46,13 @@ def _pairwise_similarity(a, b, similarity):
     return result
 
 
+@validate_params(
+    {
+        "a": [tuple],
+        "b": [tuple],
```

Comment:
The documentation is misleading. We expect at least an `array-like of shape (rows, columns)` and not a tuple. Can you also update the documentation accordingly?

Looking at the internal `check_array`, we don't support sparse matrices.

Reply:
Got it. I will do that!

Reply:
@glemaitre, I think the doc was correct. ``a`` is expected to be a tuple of 2 arrays, and same for ``b``.